pkg/registration/hub/manager.go

```diff
@@ -182,12 +190,16 @@ func (m *HubManagerOptions) RunControllerManagerWithInformers(
 		controllerContext.EventRecorder,
 	)
 
-	rbacFinalizerController := rbacfinalizerdeletion.NewFinalizeController(
+	gcController := gc.NewGCController(
```

Comment:
I think we might need a flag to let user specify a list gvr they would like to gc upon cluster deletion. This could help also to handle the addon's CRs in the cluster namespace

Reply:
added a flag named `gc-resource-list`